pom.xml

```diff
@@ -34,7 +35,7 @@
 		<repository.url>git@github.com:${project.github.repository}.git</repository.url>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<project.build.outputTimestamp>2024-01-11T15:26:09Z</project.build.outputTimestamp>
+		<project.build.outputTimestamp>${git.commit.author.time}</project.build.outputTimestamp>

```

Comment:
Nice :)

Reply:
Sadly this property is overridden with every release and must be reverted to the expression after a release is posted.

Reply:
I see - perhaps we could include that revert into the release pipeline?

Reply:
Right. It could be done with a regex. Let me have a look.